packages/nextjs/package.json

```diff
@@ -48,6 +48,7 @@
     "eslint-plugin-prettier": "^4.2.1",
     "postcss": "^8.4.16",
     "prettier": "^2.8.4",
+    "tailwind-scrollbar": "^3.0.4",
```

Comment:
Not sure about this lib. For me, if a scrollbar is used somewhere in the app, then all the scrollbars should be the same. Let's see what others think.

As I understand, it's impossible with this lib to make all scrolls the same from one place, am I right?

Reply:
I think it's fine we use the default browser instead of using `tailwind-scrollbar`, because we don't want to set any extra default styles ourselves and rather leave it to the developers whether they want to customize the scroll bar(if yes then whether they want to do it through pure css or rely on any other library)

I know it looks a bit raw on Windows but I think it's fine, not the worth for addition of a library 😅 🙌